README.md

```diff
@@ -18,8 +18,7 @@ You can find official releases of `runc` on the [release](https://github.com/ope
 
 ### Security
 
-If you wish to report a security issue, please disclose the issue responsibly
-to security@opencontainers.org.
+If you wish to report a security issue, please see the protocol under [security](security).
```

Comment:
Should be `[security]`.

Reply:
okay I think I got it right this time.

Reply:
That works, as would `[security][security]`. :P